packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagHasPrimaryAction.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Popover, PopoverTrigger, Button, PopoverSurface, makeStyles } from '@fluentui/react-components';
+import { InteractionTag } from '@fluentui/react-tags-preview';
+
+const useStyles = makeStyles({
+  popover: {
+    width: '360px',
+    maxWidth: '100%',
+    height: 'fit-content',
+  },
+  popoverHeader: {
+    display: 'flex',
+    columnGap: '10px',
+    alignItems: 'center',
+  },
+});
+
+export const HasPrimaryAction = () => {
+  const styles = useStyles();
+
+  const [dismiss, setDismiss] = React.useState(false);
+  const handleDismiss = (e: React.MouseEvent) => {
+    e.stopPropagation();
+    setDismiss(true);
+  };
+
+  return dismiss ? (
+    <Button onClick={() => setDismiss(false)}>restore tag</Button>
```

Comment:
looking at the context of this simple example, instead of actually unmounting this and having to handle the logic would it make sense to simply do an `alert` instead?